src/main.rs

```diff
@@ -28,25 +12,26 @@ fn main() {
 		Err(_)=> false,
 	};
 	if as_root {
-		println!("Warning: tray started as root! This is unsafe!");
+		println!("Warning: tray started as root! Aborting.");
+      return;
```

Comment:
why? its the user's choice

Reply:
I was never clear on why it's dangerous to run this as root, but if it is, why not just disallow it?

Reply:
> https://wiki.archlinux.org/index.php/Running_GUI_applications_as_root

just because it's dangerous we shouldn't forbid it completely, maybe the user knows what he's doing and really needs to run it as root